We know these tests fail with DT_RELR (https://sourceware.org/bugzilla/show_bug.cgi?id=28495)
so skip for now.
--- a/tests/run-elflint-self.sh
+++ b/tests/run-elflint-self.sh
@@ -18,5 +18,8 @@
 
 . $srcdir/test-subr.sh
 
+# XFAIL b/c no DT_RELR support yet (https://sourceware.org/bugzilla/show_bug.cgi?id=28495)
+exit 77
+
 testrun_on_self ${abs_top_builddir}/src/elflint --quiet --gnu-ld
 testrun_on_self_compressed ${abs_top_builddir}/src/elflint --quiet --gnu-ld
--- a/tests/run-strip-strmerge.sh
+++ b/tests/run-strip-strmerge.sh
@@ -17,6 +17,9 @@
 
 . $srcdir/test-subr.sh
 
+# XFAIL b/c no DT_RELR support yet (https://sourceware.org/bugzilla/show_bug.cgi?id=28495)
+exit 77
+
 # Generate a file with merged .shstrtab/.strtab table.
 # strip and unstrip it. Check all files with elflint.
 
diff --git a/tests/run-reverse-sections-self.sh b/tests/run-reverse-sections-self.sh
index 71afd6a..87de933 100755
--- a/tests/run-reverse-sections-self.sh
+++ b/tests/run-reverse-sections-self.sh
@@ -17,6 +17,9 @@
 
 . $srcdir/test-subr.sh
 
+# XFAIL b/c no DT_RELR support yet (https://sourceware.org/bugzilla/show_bug.cgi?id=28495)
+exit 77
+
 test_reverse_self ()
 {
   in_file="$1"
